dashboard/pages/Subscribe.py

```diff
@@ -14,7 +14,9 @@
 EMAIL_PATTERN = r'^[\w\.-]+@[\w\.-]+\.\w+$'
 
 load_dotenv()
-SNS_CLIENT = client('sns', region_name=ENV["REGION"])
+SNS_CLIENT = client(
+    'sns', region_name=ENV["REGION"],
+    aws_access_key_id=ENV["AWS_ACCESS_KEY"], aws_secret_access_key=ENV["AWS_SECRET_KEY"])
```

Comment:
Sorry about that, didn't know it needed the aws keys as well.

Reply:
no stress